client/src/Footer.js

```diff
@@ -36,6 +36,23 @@ export const Footer = () => (
               >
                 Github.
               </a>
+              {""}
+              <br />
+              Thanks to {""}
+              <a
+                style={{ textDecoration: "underline", color: "white" }}
```

Comment:
You are repeating this style several times, use a class instead and define the style in the css file

Reply:
it is done